build.gradle

```diff
@@ -2,15 +2,14 @@
 
 ext {
   // App dependencies
-  rxJavaVersion = '1.3.2'
-  rxAndroidVersion = '1.2.1'
-  kotlinVersion = '1.1.51'
-  databindingVersion = '2.3.0'
+  rxJavaVersion = '2.1.13'
+  rxAndroidVersion = '2.0.2'
+  kotlinVersion = '1.2.40'
 
-  supportLibraryVersion = '25.3.1'
+  supportLibraryVersion = '26.1.0'
```

Comment:
Would it be possible to use support lib 27.x? So that this is aligned with the main app once its migrated